### PR TITLE
Teams fx migration samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Run build pipeline before merging this PR.

File: samples/meetings-live-code-interview/nodejs/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+HTTPS=true
```
Comment (formulahendry) [inline samples/meetings-live-code-interview/nodejs/.env:1]: Is this needed?

Comment (Harikrishnan-MSFT) [reply samples/meetings-live-code-interview/nodejs/.env:1]: No needed. Fixed

**Review by formulahendry — APPROVED** [review]

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -**  
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5875107651

image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**